### PR TITLE
Add blog post on search technology meetup

### PR DESCRIPTION
**Pull request by roseritter**
@otrosien please, review my blog post. Do you still have time to comment on the blog post today, so I could publish it by tomorrow?:)

### CODE REVIEW

File: _posts/2016/2016-11-02-search-technology-meetup.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+authors: ["Rose"]
+---
+
+<style>
+.twitter-tweet {
+   margin: auto;
+}
+</style>
+
+Last friday, we hosted the seventh edition of the Search Technology Meetup Hamburg organized by [Shopping24](https://www.shopping24.de) in our new building in Holstenwall 5.
```
Comment (BirgitBader) [inline _posts/2016/2016-11-02-search-technology-meetup.md]: ... in our new office...

(the building is not ours ;-))

File: _posts/2016/2016-11-02-search-technology-meetup.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+Therefore, [Miguel Olivares](https://twitter.com/moliware) and Christian Würtz from [Xing](https://www.xing.com/) as well as Data Scientist [Andrew Clegg](https://twitter.com/andrew_clegg) from [Etsy](https://www.etsy.com/) gave us interesting insights into their search technology infrastructure, respectively.
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">The room is packed and listening to <a href="https://twitter.com/andrew_clegg">@andrew_clegg</a>! The <a href="https://twitter.com/hashtag/search?src=hash">#search</a> community is in fire in <a href="https://twitter.com/hashtag/hamburg?src=hash">#hamburg</a> at <a href="https://twitter.com/hashtag/stmhh?src=hash">#stmhh</a>. Thanks for joining us! <a href="https://t.co/1j402flFF7">pic.twitter.com/1j402flFF7</a></p>&mdash; Search Tech Meetup (@stmhamburg) <a href="https://twitter.com/stmhamburg/status/792067869114851328">October 28, 2016</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+## Evolution of search at Xing
```
Comment (BirgitBader) [inline _posts/2016/2016-11-02-search-technology-meetup.md]: This section looks too much like a "block".
Maybe you can add a paragraph inbetween, where it matches?

File: _posts/2016/2016-11-02-search-technology-meetup.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+Xing developers put special emphasis on the enhancement of phonetic search to boost the search results, even if users misspell names.
+Also, they improved the categorization of job ads.
+Thus, if users search for the English job ad or the German one for Software Developer or "Entwickler", respectively, they are now unified and result in the same search outcome, which contributes to the improvement of qualitative search.
+Furthermore, Xing developed as a service solution to what they call "search as a service ", since they offer a platform not only to search for business contacts or jobs, yet also they provide auto-suggestions based on one's personal network and geo location, etc.
+A technology that considerably contributed to Xing's success was the Lucene-based search engine [Elasticsearch](https://www.elastic.co/de/products/elasticsearch).
+
+## Search meets Data Science at Etsy
+
+In his talk, Data Scientist Andrew Clegg introduced to us the challenges Etsy.com, the marketplace for unique items and creative goods, faces with regards to search and recommendation due to the variety in inventory.
+The uniqueness of many of the offered items on Etsy adds on to this difficulty.
```
Comment (BirgitBader) [inline _posts/2016/2016-11-02-search-technology-meetup.md]: This sentence reads strange -or maybe I just don't get it?

Comment (roseritter) [inline _posts/2016/2016-11-02-search-technology-meetup.md]: @BirgitBader what exactly do you mean?

**Pull request merged**